charts/ocis/templates/appprovider/deployment.yaml

```diff
@@ -24,7 +24,7 @@ spec:
           args: ["app-provider", "server"]
           {{- include "ocis.containerSecurityContext" $ | nindent 10 }}
           env:
-            {{- include "ocis.serviceRegistry" . | nindent 12 }}
+            {{- include "ocis.serviceRegistry" $ | nindent 12 }}
```

Comment:
Why do we need this change? `.` should also be fine given that only values need to be addressed.

Reply:
Ah right. Haven't seen that this was in appprovider. Yes this one is special.